build-js.sh

```diff
@@ -579,10 +583,5 @@ export_function "_BinaryenSetAPITracing"
   -Isrc/ \
   -s EXPORTED_FUNCTIONS=[${EXPORTED_FUNCTIONS}] \
   -o bin/binaryen${OUT_FILE_SUFFIX}.js \
-  --post-js src/js/binaryen.js-post.js \
```

Comment:
have we decided that MODULARIZE_INSTANCE is the way to go here?

I'm not objecting, just I thought it was still being discussed, but I could be wrong.

In any case, is it worth undoing this, this re-doing it all with MODULARIZE_INSTANCE?

Reply:
Possibly not when you are asking. Though, previously, the global var was a singleton instance (through concatenation of `Binaryen = Binaryen()`) while module loaders received a factory function. My intention here was to make sure that module loaders receive the singleton instance as well so it's consistent.

Reply:
Yeah, consistency is important, I agree. But I can open a PR for MODULARIZE_INSTANCE later today, so it shouldn't be long before we can use it. If you don't mind undoing the changes here after that, though, then it doesn't matter either way, I just thought it might be efficient to wait.

Reply:
Sure, is fine for me either way.